src/storage/mod.rs

```diff
@@ -1452,6 +1452,18 @@ impl<E: Engine, L: LockManager, F: KvFormat> Storage<E, L, F> {
                 )?;
                 check_key_size!(keys, self.max_key_size, callback);
             }
+            Command::AcquirePessimisticLockResumed(AcquirePessimisticLockResumed {
```

Comment:
Should we make the command name consistent with the "wakeup mode" in kv proto in the later pr?

Reply:
Sorry but I didn't quite understand. What's your suggested name of this command?

Reply:
Since it seems we're not going to use the "resume" here(https://github.com/pingcap/kvproto/pull/872/files), should we change the name here to "Force" too?

Reply:
This command is not used at first. When a request arrives TiKV, it's handled in `AcquirePessimisticLock` firstly; if it's blocked by lock, after waiting and woken up, this new command will be used. Therefore I think `resumed` is a proper word for it, but the behavior described by the word `Force` (which I think means the same thing as allow_lock_with_conflict) exists in both `AcquirePessimisticLock` and this new command.